<launch>
    <!-- this simulation publish 2 pointCloud2; velodyne_points and velodyne_points2 -->


    <node pkg="pcl_ros" type="pointcloud_to_pcd" name="pointcloud_to_pcd" output="screen">
        <remap from="input" to="/laser_pointcloud_assembler"/>
        <!-- prefix : set output folder path. without this, save pcd files in.ros folder. -->
        <param name="prefix" value="/home/aisl/catkin_ws/src/latest_crane/laser_to_pcl/src/" />
        <param name="fixed_frame" value="base_link" />
        <param name="binary" value="false" />
        <param name="compressed" value="false" />
    </node>
</launch>
